apps/docs/src/content/docs/guide/oauth.md

```diff
@@ -0,0 +1,105 @@
+
+## Exchanging tokens
+
+If you've followed the previous section on getting authorization, you should now have an authorization code. Before you can access the API, you need to exchange this code for an access token. This is done by sending a POST request to the exchange token endpoint, `https://api.modrinth.com/_internal/oauth/token`. This request has to be of type urlencoded form. Make sure the `Content-Type` header is set to `application/x-www-form-urlencoded`. To authenticate this request you need to place your client secret in the `Authorization` header.
+
+In the body use these fields:
+
+| Field          | Description                                                  |
+|----------------|--------------------------------------------------------------|
+| `code`         | The authorization code                                       |
+| `client_id`    | Your client id, the same as in the authorization request     |
+| `redirect_uri` | The redirect URI which was redirected to after authorization |
+| `grant_type`   | Always `authorization_code` in Modrinth                      |
+
+If the request succeeds, you should receive a JSON payload with these fields:
+
+| Field          | Description                                          |
+|----------------|------------------------------------------------------|
+| `access_token` | The access token you can use to access the API       |
+| `token_type`   | Currently only `Bearer`                              |
+| `expires_in`   | The amount of seconds until the access token expires |
+
+To use this access token, you attach it to API requests in the `Authorization` header prefixed by `Bearer`. For example `Authorization: Bearer mro_somesecretcharacters`. To get basic information about the authorizer, you can use the [`/user` endpoint], which automatically gets the user from the header.
+
+## FAQ
+
+**Does Modrinth support implicit authorization grant?** No, Modrinth doesn't currently support implicit authorization grants. Only code grants are supported. Implicit authorization is an alternative way of receiving access rights for users, without the need for a backend server.
+
+**Does Modrinth use JWTs?** No, Modrinth doesn't use JSON Web Tokens for authentication. Instead, tokens starting with `mra_` (standard session), `mrp_` (personal access token) and `mro_` (application) are used.
+
+**Is Modrinth compatible with OpenId Connect?** No, Modrinth doesn't implement the OpenId Connect protocol. The key differences between OAuth and OpenId Connect are usage of JSON and JWTs.
+
+**Is there a token revocation endpoint?** Modrinth's API does currently not have an endpoint for the application to deauthorize itself.
+
+**I have a question!** Questions are very much welcome, and we encourage you to ask in #api-development in the [Discord guild], or create an issue on [GitHub].
```

Comment:
Not sure if any of this is necessary besides saying we have api-development channel and support website